docs/_docs/community/community.md

```diff
@@ -16,7 +16,7 @@ If you're looking for support for Jekyll, there are a lot of options:
 
 * Read the [Jekyll Documentation]({{ '/docs/' | relative_url }})
 * If you have a question about using Jekyll, start a discussion on the [Jekyll Forum](https://talk.jekyllrb.com/) or [StackOverflow](https://stackoverflow.com/questions/tagged/jekyll)
-* Chat with Jekyllers &mdash; Join our [Gitter channel](https://gitter.im/jekyll/jekyll) or our [IRC channel #jekyll on Freenode](irc://irc.freenode.net/#jekyll)
+* Chat with Jekyllers &mdash; Join our [Gitter channel](https://gitter.im/jekyll/jekyll) or our IRC channels #jekyll on [Libera](irc://irc.libera.chat/#jekyll) or [Freenode](irc://irc.freenode.net/#jekyll).
```

Comment:
```suggestion
* Chat with Jekyllers &mdash; Join our [Gitter channel](https://gitter.im/jekyll/jekyll) or our IRC channels #jekyll on [Libera](irc://irc.libera.chat/#jekyll).
```

I feel after the most recent event in the Freenode world, we should take this opportunity to abandon Freenode entirely.

> [keitwo] [Global Notice 1/3] We are moving past legacy freenode to a new fork. The new freenode is launched. You will slowly be disconnected and when you reconnect, you will be on the new freenode. We patiently await to welcome you in freedom's holdout - the freenode.
> [keitwo] [Global Notice 2/3] If you're looking to connect now, you can already /server chat.freenode.net 6697 (ssl) or 6667 (plaintext). It's a new genesis for a new era. Thank you for using freenode, and Hello World, from the future. freenode is IRC. freenode is FOSS. freenode is freedom.
> [keitwo] [Global Notice 3/3] When you connect, register your nickname and your channel and get started. It's a new world. We're so happy to welcome you and the millions of others. We will be posting more information in the coming days on our website and twitter. Otherwise, see you on the other side!

By the sounds of it, we literally need to re-register our accounts _and_ channels???

Reply:
Initially I was strongly against abandoning Freenode and struggling with the idea because Freenode has provided us with a safe, friendly, useful IRC channel to come together, talk, learn, and share jekyll knowledge.

I put some time between @allejo comment and suggestion in order to dig around on Freenode to determine future viability.

During this time I received positive helpfulness from a  number of [presumably-temporary] staff operators. That was refreshing and renewing.

Unfortunately, also during this time, I was also abused, name-called (of which I will not repeat here as it 100% blatantly violates our (and likely anyones) Code of Conduct).

I could go on with details, quotes, etc. but it is not necessary unfortunately.

As there are active users existing on both networks, I plan on keeping the Freenode #jekyll channel operational and functional.

I strongly suggest we remove references to Freenode from the site, docs, and certainly as a pointer to join IRC to get help and collaboration. It is [currently] an unsafe, unfriendly, inappropriate, and opposite our Code of Conduct.

The Libera network should be the only reference; Freenode should be purged and avoided unfortunately.

@allejo - will you please edit your changes to adapt "channels" to "channel", etc.?

I will then be in support and encouragement of this change.

Thanks to the legacy Freenode spirit and people we once knew and respected and appreciated.

Reply:
I've updated the suggestion to be singular; good catch!